build.gradle.kts

```diff
@@ -44,28 +43,28 @@ kotlin {
         }
     }
 
-    sourceSets {
-        val commonMain by getting {
-            // SQLDelight ORM will be generated here
+    sourceSets { 
+        commonMain {
+            dependencies {
+                implementation("app.softwork:postgres-native-sqldelight-driver:$psql_driver_version")
```

Comment:
at the moment, you put the sq files into common main, so the driver is needed at commonmain too